### ISSUE TITLE
Offline translation?

### ISSUE THREAD

**Issue by zubairahmed-ai** [Helsinki-NLP/OPUS-MT-train#60]
Hi 

I need English to Urdu model to run offline, it downloads from `https://huggingface.co/api/models/Helsinki-NLP/opus-mt-en-ur` 

Is it possible to install and run this completely offline from the file system?

**Comment by jorgtied** [comment]
Yes, you can. Follow the instructions and examples from the huggingface transformers library. Alternatively, you can also use the original model from OPUS-MT and MarianNMT.
